config/development.js

```diff
@@ -10,6 +10,7 @@ module.exports = {
   port: port,
   enableFileLogs: false,
   enableConsoleLogs: true,
+  discordNewComersChannelId: "709080951824842783",
```

Comment:
Should this come from env?

Reply:
not actually, it is okay to have it this way, we can get it easily anyway